### PR TITLE
Adding Drizly to workflow menu

### PR DESCRIPTION
**Pull request by emily-hawkins**
Added docs for drizly's Github actions workflow

### CODE REVIEW

File: README.md
Comment (barrywhart) [inline README.md:4]: Attention to detail = 💯 🎉

File: menu_of_workflows/drizly/README.md
Comment (barrywhart) [inline menu_of_workflows/drizly/README.md]:
```suggestion
This action will take any changed files (modified or added) in a PR and lint them using your SQLFluff configuration.
```

**Review by barrywhart — APPROVED** [review]
Looks great!

I noticed one little typo we may want to fix before merging.

File: menu_of_workflows/drizly/README.md
Comment (emily-hawkins) [reply menu_of_workflows/drizly/README.md]: fixed! thank you

**Comment by emily-hawkins** [comment]
i dont have write access to this repo, so whoever can, feel free to merge when you feel good about this!

**Pull request merged**
